contents/tutorials/next-steps-after-installing.md

```diff
@@ -0,0 +1,160 @@
+  properties: {
+    movie_id: 'Return of the Hedgehogs',
+    category: 'thriller'
+  }
+})
+```
+
+```node
+client.capture({
+  distinctId: 'ian@posthog.com',
+  event: 'movie played',
+  properties: {
+      movieId: 'Return of the Hedgehogs',
+      category: 'thriller',
+  },
+})
+```
+
+```python
+posthog.capture(
+  'ian@posthog.com', 
+  'movie played', 
+  {
+    'movie_id': 'Return of the Hedgehogs',
+    'category': 'thriller'
+  }
+)
+```
+
+```go
+client.Enqueue(posthog.Capture{
+  DistinctId: "ian@posthog.com",
+  Event:      "movie played",
+  Properties: posthog.NewProperties().
+    Set("movieId", "Return of the Hedgehogs").
+    Set("category", "thriller"),
+})
+```
+
+</MultiLanguage>
+
+To figure out where to capture events, think about the areas that have the biggest impact on the success of your product. We provide some ideas in “[finding your North Star metric and why it matters](/blog/north-star-metrics).” Some recommendations for areas you should focus on capturing:
+
+- Signup
+- Subscription
+- Core features
+- New features
+- Marketing website
+
+Autocapture covers some of these already. In this case, you can create actions (a combination of one or more events) to better represent user behavior. You can create actions in the “data management” section of the sidebar or interactively using the toolbar (click “launch toolbar” on the sidebar).
+
+![Action](../images/tutorials/next-steps-after-installing/action.png)
+
+Be sure to add properties to your events to better capture the variables related to the events such as the value, type, version, or something else. For example, you might want to know what version of the product someone is using or if they are a customer or not. Properties can help you here, and you can set them for users as well. 
+
+Finally, you may also want to connect to external services like [Stripe](/tutorials/stripe-payment-data) or [HubSpot](/apps/hubspot-connector) to get data. You can do that using [apps](/apps). To install an app, search for it in “browse apps,” then configure and activate it.
+
+## Building a dashboard
+
+Once you’ve got events flowing in, you’ll want to use that data to better understand your product. Instead of creating insights one at a time, you should think about creating a dashboard of multiple insights. Keep in mind what data you are capturing when you do this.
+
+Some recommendations include:
+
+- Key product analytics: usage trends, feature conversion, daily/weekly active users, retention
+- Website traffic: overall traffic, top-performing pages, traffic breakdowns, sources
+- AARRR: signup trends, usage trends, customer retention, revenue growth trends, attribution table
+- Company Scorecard: key metric trends, signups, success rate trends, conversion funnels, revenue
+
+> For both product analytics and website traffic, you can use templates to shortcut their creation. When creating a dashboard, select either option from the “template” dropdown.
```

Comment:
```suggestion
> Want to create dashboards quickly? PostHog offers several templates to cover common areas of interest, such as website traffic. To use a template, select an option from the "Template" dropdown when creating a new dashboard.
```

We have 2x more templates about to launch, as well as the option to make new templates easily (which will mean even more templates). To futureproof, suggest changing this.